### ISSUE TITLE
broken link in README.bs2000

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#11052]
Migrated from [rt.perl.org#82476](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=82476) (status was 'resolved')

Searchable as RT82476$

**Comment by p5pRT** [comment]
### From @mstevens

#### Created by @mstevens



The following files appear to contain a broken link&#8203;:

mstevens@&#8203;mstevens\-desktop&#8203;:\~/bleadperl/perl$ fgrep 'http&#8203;://lists\.perl\.org/showlist\.cgi?name=perl\-mvs' \*
README\.bs2000&#8203;:    http&#8203;://lists\.perl\.org/showlist\.cgi?name=perl\-mvs
README\.os390&#8203;:    http&#8203;://lists\.perl\.org/showlist\.cgi?name=perl\-mvs
README\.vmesa&#8203;:    http&#8203;://lists\.perl\.org/showlist\.cgi?name=perl\-mvs

I get a server error when attempting to go to this URL\.

\(Filing somewhat as an experiment\, I have plenty more broken links if anyone knows what to do with them\.\.\.\)

<details><summary>Perl Info</summary>

```

Flags:
    category=docs
    severity=low

Site configuration information for perl 5.13.8:

Configured by mstevens at Wed Jan 19 12:45:37 GMT 2011.

Summary of my perl5 (revision 5 version 13 subversion 8) configuration:
  Commit id: 1b9043bb5ee28cabe60d5dc8a9c4365e365c7ca0
  Platform:
    osname=linux, osvers=2.6.35.4, archname=x86_64-linux
    uname='linux eris 2.6.35.4 #4 smp tue sep 21 09:54:22 bst 2010 x86_64 gnulinux '
    config_args='-des -Dusedevel -Dprefix=/home/mstevens/bleadperl/build'
    hint=previous, useposix=true, d_sigaction=define
    useithreads=undef, usemultiplicity=undef
    useperlio=define, d_sfio=undef, uselargefiles=define, usesocks=undef
    use64bitint=define, use64bitall=define, uselongdouble=undef
    usemymalloc=n, bincompat5005=undef
  Compiler:
    cc='cc', ccflags ='-fno-strict-aliasing -pipe -fstack-protector -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64',
    optimize='-O2',
    cppflags='-fno-strict-aliasing -pipe -fstack-protector -I/usr/local/include -fno-strict-aliasing -pipe -fstack-protector -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64'
    ccversion='', gccversion='4.3.2', gccosandvers=''
    intsize=4, longsize=8, ptrsize=8, doublesize=8, byteorder=12345678
    d_longlong=define, longlongsize=8, d_longdbl=define, longdblsize=16
    ivtype='long', ivsize=8, nvtype='double', nvsize=8, Off_t='off_t', lseeksize=8
    alignbytes=8, prototype=define
  Linker and Libraries:
    ld='cc', ldflags =' -fstack-protector -L/usr/local/lib'
    libpth=/usr/local/lib /lib /usr/lib /lib64 /usr/lib64
    libs=-lnsl -ldb -ldl -lm -lcrypt -lutil -lc
    perllibs=-lnsl -ldl -lm -lcrypt -lutil -lc
    libc=/lib/libc-2.7.so, so=so, useshrplib=false, libperl=libperl.a
    gnulibc_version='2.7'
  Dynamic Linking:
    dlsrc=dl_dlopen.xs, dlext=so, d_dlsymun=undef, ccdlflags='-Wl,-E'
    cccdlflags='-fPIC', lddlflags='-shared -O2 -L/usr/local/lib -fstack-protector'

Locally applied patches:
    


@INC for perl 5.13.8:
    /home/mstevens/bleadperl/build/lib/site_perl/5.13.8/x86_64-linux
    /home/mstevens/bleadperl/build/lib/site_perl/5.13.8
    /home/mstevens/bleadperl/build/lib/5.13.8/x86_64-linux
    /home/mstevens/bleadperl/build/lib/5.13.8
    .


Environment for perl 5.13.8:
    HOME=/home/mstevens
    LANG=en_GB.UTF-8
    LANGUAGE (unset)
    LD_LIBRARY_PATH (unset)
    LOGDIR (unset)
    PATH=/home/mstevens/bin:/home/mstevens/bin:/usr/local/bin:/usr/bin:/bin:/usr/bin/X11:/usr/games:/usr/local/gale/bin:/sbin:/usr/sbin:/usr/local/gale/bin:/sbin:/usr/sbin
    PERL_BADLANG (unset)
    SHELL=/usr/bin/zsh

```
</details>

**Comment by p5pRT** [comment]
### From @jkeenan

On Wed Jan 19 08&#8203;:19&#8203;:13 2011\, mstevens wrote&#8203;:

> The following files appear to contain a broken link&#8203;:
> 
> mstevens@&#8203;mstevens\-desktop&#8203;:\~/bleadperl/perl$ fgrep
> 'http&#8203;://lists\.perl\.org/showlist\.cgi?name=perl\-mvs' \*
> README\.bs2000&#8203;:    http&#8203;://lists\.perl\.org/showlist\.cgi?name=perl\-mvs
> README\.os390&#8203;:    http&#8203;://lists\.perl\.org/showlist\.cgi?name=perl\-mvs
> README\.vmesa&#8203;:    http&#8203;://lists\.perl\.org/showlist\.cgi?name=perl\-mvs
> 
> I get a server error when attempting to go to this URL\.
> 

For me\, that URL resolves to&#8203;:

http&#8203;://lists\.perl\.org/list/perl\-mvs\.html

\.\.\. which displays as expected\.

Can someone else confirm so that we can close this RT?  \(If you have
other bad links\, please open new bug tickets\.\)

Thank you very much\.
Jim Keenan

**Comment by p5pRT** [comment]
The RT System itself - Status changed from 'new' to 'open'

**Comment by p5pRT** [comment]
### From @cpansprout

On Thu Dec 08 17&#8203;:06&#8203;:25 2011\, jkeenan wrote&#8203;:
> On Wed Jan 19 08&#8203;:19&#8203;:13 2011\, mstevens wrote&#8203;:
> 
> > The following files appear to contain a broken link&#8203;:
> > 
> > mstevens@&#8203;mstevens\-desktop&#8203;:\~/bleadperl/perl$ fgrep
> > 'http&#8203;://lists\.perl\.org/showlist\.cgi?name=perl\-mvs' \*
> > README\.bs2000&#8203;:    http&#8203;://lists\.perl\.org/showlist\.cgi?name=perl\-mvs
> > README\.os390&#8203;:    http&#8203;://lists\.perl\.org/showlist\.cgi?name=perl\-mvs
> > README\.vmesa&#8203;:    http&#8203;://lists\.perl\.org/showlist\.cgi?name=perl\-mvs
> > 
> > I get a server error when attempting to go to this URL\.
> > 
> 
> For me\, that URL resolves to&#8203;:
> 
> http&#8203;://lists\.perl\.org/list/perl\-mvs\.html
> 
> \.\.\. which displays as expected\.
> 
> Can someone else confirm so that we can close this RT?  \(If you have
> other bad links\, please open new bug tickets\.\)

Works for me\.

\-\- 

Father Chrysostomos

**Comment by p5pRT** [comment]
@cpansprout - Status changed from 'open' to 'resolved'

**Comment by p5pRT** [comment]
### From @mstevens

On Thu\, Dec 08\, 2011 at 10&#8203;:39&#8203;:15PM \-0800\, Father Chrysostomos via RT wrote&#8203;:
> On Thu Dec 08 17&#8203;:06&#8203;:25 2011\, jkeenan wrote&#8203;:
> > On Wed Jan 19 08&#8203;:19&#8203;:13 2011\, mstevens wrote&#8203;:
> > 
> > > The following files appear to contain a broken link&#8203;:
> > > 
> > > mstevens@&#8203;mstevens\-desktop&#8203;:\~/bleadperl/perl$ fgrep
> > > 'http&#8203;://lists\.perl\.org/showlist\.cgi?name=perl\-mvs' \*
> > > README\.bs2000&#8203;:    http&#8203;://lists\.perl\.org/showlist\.cgi?name=perl\-mvs
> > > README\.os390&#8203;:    http&#8203;://lists\.perl\.org/showlist\.cgi?name=perl\-mvs
> > > README\.vmesa&#8203;:    http&#8203;://lists\.perl\.org/showlist\.cgi?name=perl\-mvs
> > > 
> > > I get a server error when attempting to go to this URL\.
> > > 
> > 
> > For me\, that URL resolves to&#8203;:
> > 
> > http&#8203;://lists\.perl\.org/list/perl\-mvs\.html
> > 
> > \.\.\. which displays as expected\.
> > 
> > Can someone else confirm so that we can close this RT?  \(If you have
> > other bad links\, please open new bug tickets\.\)
> 
> Works for me\.

I agree it is currently loading\. Perhaps it has been fixed\!

Michael

**Comment by p5pRT** [comment]
### From @smpeters

On Fri\, Dec 9\, 2011 at 5&#8203;:09 AM\, Michael Stevens \<mstevens@&#8203;etla\.org> wrote&#8203;:
> On Thu\, Dec 08\, 2011 at 10&#8203;:39&#8203;:15PM \-0800\, Father Chrysostomos via RT wrote&#8203;:
>> On Thu Dec 08 17&#8203;:06&#8203;:25 2011\, jkeenan wrote&#8203;:
>> > On Wed Jan 19 08&#8203;:19&#8203;:13 2011\, mstevens wrote&#8203;:
>> >
>> > > The following files appear to contain a broken link&#8203;:
>> > >
>> > > mstevens@&#8203;mstevens\-desktop&#8203;:\~/bleadperl/perl$ fgrep
>> > > 'http&#8203;://lists\.perl\.org/showlist\.cgi?name=perl\-mvs' \*
>> > > README\.bs2000&#8203;:    http&#8203;://lists\.perl\.org/showlist\.cgi?name=perl\-mvs
>> > > README\.os390&#8203;:    http&#8203;://lists\.perl\.org/showlist\.cgi?name=perl\-mvs
>> > > README\.vmesa&#8203;:    http&#8203;://lists\.perl\.org/showlist\.cgi?name=perl\-mvs
>> > >
>> > > I get a server error when attempting to go to this URL\.
>> > >
>> >
>> > For me\, that URL resolves to&#8203;:
>> >
>> > http&#8203;://lists\.perl\.org/list/perl\-mvs\.html
>> >
>> > \.\.\. which displays as expected\.
>> >
>> > Can someone else confirm so that we can close this RT?  \(If you have
>> > other bad links\, please open new bug tickets\.\)
>>
>> Works for me\.
>
> I agree it is currently loading\. Perhaps it has been fixed\!
>
> Michael

I've just updated the assorted Pods with the new URLs along with
cleaning the link to the ithreads list\.  Also\, updated the URL in
Porting/checkURL\.pl\.

Steve Peters
steve@&#8203;fisharerojo\.org
